ansible_collections/arista/avd/roles/eos_designs/python_modules/network_services/vrfs.py

```diff
@@ -45,8 +45,14 @@ def vrfs(self) -> list | None:
                 new_vrf = {
                     "name": vrf_name,
                     "tenant": tenant["name"],
-                    "ip_routing": True,
                 }
+
+                # MLAG IBGP Peering VLANs per VRF
+                if self._mlag_ibgp_peering_enabled(vrf, tenant):
```

Comment:
This is not the correct condition. You should be looking for `overlay_mlag_rfc5549`